app/helpers/filters_helper.rb

```
@@ -11,11 +19,15 @@ def search_path(type)
         when 'Parameter'
           '' # parameter is only used in API
         else
-          resource_path(type) + '/auto_complete_search'
+          resource_path(type) + auto_complete_search_path
```

Disagree. Silent failures, esp. in this place (due to ajax) will be harder to troubleshoot.